specification/resources/resource-manager/Microsoft.Resources/stable/2020-10-01/subscriptions.json

```diff
@@ -0,0 +1,730 @@
+          "default": {
+            "description": "Error response saying resource name is invalid",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Check validity for a resource name.": {
+            "$ref": "./examples/CheckResourceName.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PairedRegion": {
+      "description": "Information regarding paired region.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the paired region."
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The fully qualified ID of the location. For example, /subscriptions/00000000-0000-0000-0000-000000000000/locations/westus."
+        },
+        "subscriptionId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription ID."
+        }
+      }
+    },
+    "LocationMetadata": {
+      "description": "Location metadata information",
+      "properties": {
+        "regionType": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of the region.",
+          "enum": [
+            "Physical",
+            "Logical"
+          ],
+          "x-ms-enum": {
+            "name": "RegionType",
+            "modelAsString": true
+          }
+        },
+        "regionCategory": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The category of the region.",
+          "enum": [
+            "Recommended",
+            "Extended",
+            "Other"
+          ],
+          "x-ms-enum": {
+            "name": "RegionCategory",
+            "modelAsString": true
+          }
+        },
+        "geographyGroup": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The geography group of the location."
+        },
+        "longitude": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The longitude of the location."
+        },
+        "latitude": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The latitude of the location."
+        },
+        "physicalLocation": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The physical location of the Azure location."
+        },
+        "pairedRegion": {
+          "type": "array",
+          "description": "The regions paired to this region.",
+          "items": {
+            "$ref": "#/definitions/PairedRegion"
+          }
+        },
+        "homeLocation": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The home location of an edge zone."
+        }
+      }
+    },
+    "Location": {
+      "properties": {
+        "id": {
```

Comment:
Is Location a resource?  If so, it should extend one of the common resource types.  Also, does it have a 'systemData' property?

Reply:
Location is not a resource as of today, although we do have this "id" property which put here for future extensibility.